app/models/concerns/bulkrax/has_matchers.rb

```diff
@@ -86,9 +86,13 @@ def get_object_name(field)
     # @return [Array] hyrax fields
     def field_to(field)
       fields = mapping&.map do |key, value|
-        value['from'] = value['from'].join(', ') if value['from'].instance_of?(Array)
+        return unless value
 
-        key if (value.present? && value['from']&.include?(field)) || key == field
+        if value['from'].instance_of?(Array)
```

Comment:
@orangewolf using a guard clause above instead of `if value&.[]('from').instance_of?(Array)` on line 91 because rubocop didn't like that. offense: "Do not chain ordinary method call after safe navigation operator."

 plus it reads a bit cleaner this way.